src/Tribe/Admin/Activation_Page.php

```diff
@@ -187,7 +187,7 @@ public function maybe_redirect() {
 
 		delete_transient( $this->args['activation_transient'] );
 
-		if ( ! current_user_can( Tribe__Settings::instance()->requiredCap ) ) {
+		if ( ! current_user_can( tribe( 'settings' )->requiredCap ) ) {
```

Comment:
I like the shift to using `tribe()`, but there's one thing that I feel should be mitigated.

Technically, there is a chance that calling `tribe()` could result in an exception being thrown. I admit, this chance is very small. But since this will run in an unpredictable WordPress environment, I think we should be extra cautious.

This applies to all the places it is called.

Reply:
@Camwyn Any thoughts on this? I know it's not exactly unique to your PR but I thought it might be worth discussing or thinking about.

Reply:
In the old code - when `Tribe__Settings::instance()` was called it returned `tribe( 'settings' )` so I think if that was going to happen it already would be happening.

I understand and appreciate the concern though.

If it does happen somewhere, we've either got a build or timing issue (that should be addressed anyway).